Marlin/Configuration.h

```diff
@@ -2022,7 +2011,7 @@
  *  - Change to green once print has finished
  *  - Turn off after the print has finished and the user has pushed a button
  */
-#if ENABLED(BLINKM) || ENABLED(RGB_LED) || ENABLED(RGBW_LED) || ENABLED(PCA9632) || ENABLED(PCA9533)|| ENABLED(NEOPIXEL_LED)
+#if ENABLED(BLINKM) || ENABLED(RGB_LED) || ENABLED(RGBW_LED) || ENABLED(PCA9632) || ENABLED(NEOPIXEL_LED)
```

Comment:
I'd like to keep all the above changes starting with "`// 2-wire Non-latching LCD`…". (A "git rebase" to the latest bugfix-2.0.x would've avoided needing to fix these throughout the PR. Possibly a rebase now will still work, so I'll try that.)